client/src/views/components/Header/MonthController/MonthController.ts

```diff
@@ -0,0 +1,93 @@
+import { Arrow, MonthControlImage } from '@src/static/image-urls';
+import { Direction, HTMLText, Month, Year } from '@src/types';
+import handleEvent from '@src/utils/handleEvent';
+import { $, createDOMWithSelector, monthText } from '@src/utils/helper';
+
+import './MonthController.scss';
+
+export default class MonthController {
+  $MonthController: HTMLElement;
```

Comment:
변수명 컨벤션 실수하신부분이 있으신거같습니다!
```suggestion
  $monthController: HTMLElement;
```